apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -44,6 +44,36 @@ class DataVisualizer extends React.Component {
     this.setState({...INITIAL_STATE});
   };
 
+  createPivotTable = (records, rowName, columnName) => {
+    let countMap = {};
+
+    // Find all values in columnName - these will be the columns of the pivot table
+    let pivotedColumns = new Set(_.map(records, record => record[columnName]));
```

Comment:
I only suggested `_.map` earlier this week because it's a convenient way to iterate over an object.  If `records` is an array, I've got a mild preference for using the built-in `Array.map`.  Also, I'm pretty sure `pivotedColumns` can be `const` since you never reassign it.
```suggestion
    const pivotedColumns = new Set(records.map(record => record[columnName]));
```